Resources/Prototypes/Recipes/Construction/Graphs/conveyor.yml

```diff
@@ -6,7 +6,9 @@
       edges:
         - to: entity
           steps:
-            - component: ConveyorAssembly
+            # Pretty sure this doesn't work but i'm waiting for construction tags anyway *dab dab*
+            - prototype: ConveyorAssembly
+              amount: 1
```

Comment:
`amount` doesn't exist (and after thinking about it, it probably never will as it would overcomplicate the current construction logic--)
```suggestion
```